common/app/views/fragments/newHeader.scala.html

```diff
@@ -27,41 +39,44 @@
                     ))
                 </a>
 
-                <div class="new-header__cta-container hide-until-mobile">
+                <div class="new-header__top-bar hide-until-mobile">
                     @defining(
                         Edition(request).id.toLowerCase()
                     ) { editionId =>
                       @if(!page.metadata.hasSlimHeader) {
-                            <a class="header-cta-item header-cta-item--primary"
+                            <a class="top-bar__item top-bar__item--cta js-change-become-member-link"
                               data-link-name="nav2 : supporter-cta"
                               data-edition="@{editionId}"
                               href="@getContributionOrSupporterUrl(editionId)">
-                              <span class="header-cta-item__label">
+                              <span>
                                   @if(editionId == "us") {
-                                      make a <span class="header-cta-item__new-line">contribution</span>
+                                     make a <br>contribution
```

Comment:
would you want this to break with no other style applied?

Reply:
Yep!

Reply:
won't this wrap anyway?

Reply:
Not unless I gave it a fixed width/max-width, and I want the width to be flexible to accommodate the different us/uk messaging.